### ISSUE TITLE
covid-19: Data exports are not working

### ISSUE THREAD

**Issue by boulund** [ctmrbio/claritylims#205]
On 2020-07-16 it was discovered that the `get_all_samples.py` script stopped working. Likely caused by too many samples. 
Some troubleshooting by me found that it was unlikely that we would get it to work again, as it relies on the genologics Python package and it would require a lot of work to circumvent our issues. We decided to hardcode a SQL query to fix it. @chichackles has taken care of that.

I also took the opportunity to verify that the `datagraphics_parser.py` works as intended with the new output produced by the SQL query. I found some issues, but have worked around them, PR incoming.

@chichackles will add the required SQL command to the repo so we can put it in a cronjob. For it to run as the `glsai` user as a cronjob, a `~/.pgpass` file is required, with the format:
```
localhost:5432:ClarityDB:clarity:<password>
```
Then the command to run the SQL query is simply:
```
psql --no-align --field-separator=","  --output=/tmp/report_$(date +%Y%m%d) --file /path/to/reportCOVID19.sql ClarityDB clarity
```

**Comment by boulund** [comment]
I've now validated the export script on stage. Will merge from the branch @chichackles put the code in to the `prod` branch as well.

**Comment by boulund** [comment]
@chichackles I didn't notice it before during the PR, but it seems you accidentally created branch `Issue-#205` based off `dev` instead of `prod`, which means it's impossible to merge that branch into `prod` without including all the in-progress commits currently in `dev`. This is not what we want right now. 

I worked around it by manually cherry-picking the commits we made to the `Issue-#205` branch and committed them directly to `prod`, so no worries!
